service_info/static/less/base/base.less

```diff
@@ -61,3 +61,9 @@ img {
 *[dir="LTR"] {
   text-align: left;
 }
+
+table {
+  @media (max-width: 992px) {
```

Comment:
Curious:   I'm accustomed to seeing the ``max-width`` values one less than the ``min-width`` values for the same breakpoint.  It seems that we use 992 always; shouldn't max be 991, or are all these things (below) conceptually different?  (no 991 or 993 in our ``.less``)

```
./service_info/static/less/content-types/homepage.less:  @media (min-width: 992px) {
./service_info/static/less/aldryn/aldryn-newsblog/includes/article-card.less:  @media (max-width: 992px) {
./service_info/static/less/base/base.less:  @media (min-width: 992px) {
./service_info/static/less/base/header/brand.less:    @media (max-width: 992px) {
./service_info/static/less/base/header/language-picker.less:    @media (max-width: 992px) {
```